src/main/java/actions/TaskList.java

```diff
@@ -0,0 +1,98 @@
+package actions;
+
+import tasks.Task;
+
+import java.util.ArrayList;
+
+public class TaskList {
+    private ArrayList<Task> taskArrayList;
+    String line = "~~*~~*~~*~~*~~*~~*~~*~~*~~*~~\n";
```

Comment:
This line seems to be a constant, perhaps change it to SCREAMING_SNAKE_CASE. Also, do add access modifiers to it.
perhaps
`private final String LINE = "~~*~~*~~*~~*~~*~~*~~*~~*~~*~~\n";`